Homebrew/emacs_formula.rb

```diff
@@ -81,7 +81,7 @@ def ert_run_tests(*files)
 
   ### Slightly better tested than ert_run_tests
   def byte_compile(*files)
-    emacs_args = %W[ --batch -Q ]
```

Comment:
I don't know the definitive answer, but this [EmacsWiki page](https://www.emacswiki.org/emacs/BatchMode) says it doesn't:

> Emacs batch mode does not read your personal config (or any config, it’s equivalent to -Q) so you don’t get packages loaded.